crashreports/crashreporter/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+#Adds a Crash Reporting dialog which may be invoked by a crashing program
```

Comment:
Comment should emphasize that this builds a separate executable, invoked by fork-exec or Windows equivalent, when the main program is trapping a crash.